docbook/Admin_Guide/en-US/config/misc.xml

```diff
@@ -121,6 +121,33 @@
 			</listitem>
 		</varlistentry>
 		<varlistentry>
+			<term>$g_public_urls</term>
+			<listitem>
+				<para>Enable usage of public share URLs. 
+					The default setting is OFF.
+				</para>
+				<para>Enabling this option will show additional "Public Share
+					URL" read-only field when viewing the bug (in "Users 
+					monitoring this issue" section) as well as additional 
+					"token" field in SOAP API. The URL can be copied, 
+					used and shared. 
+				</para>
+				<para>Access via public share URL does not require loggin in
+					(e.g. access is anonymous). This option does not 
+					require $g_allow_anonymous_login to be enabled.
+				</para>
+				<para>When user visits public share URL - he will see a limited
+					read-only view of the bug with "Full View" button asking
+					to log in. User may see current status of the bug and
+					minor details (such as modification date). Public bugs
+					will show summary, description and steps to reproduce in
+					limited view. Private bugs will show only summary, but no
```

Comment:
The limited view page seems to be missing interesting information to get the full status of an issue including:
- Status
- Public notes

Do we need something similar to `$g_bug_view_page_fields` and flags to control aspects like showing public notes.